tests/Support/Assert.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Widgets\Tests\Support;
+
+use PHPUnit\Framework\TestCase;
+use ReflectionClass;
+use ReflectionException;
+use ReflectionObject;
+
+use function str_replace;
+
+/**
+ * @psalm-suppress PropertyNotSetInConstructor
+ */
+final class Assert extends TestCase
```

Comment:
Suggest to either add this to a trait similar to `TestTrait` or directly extend from class (rename it then). A mix of instance / static assert methods is a bit confusing.

Reply:
OK, can be done in a separate PR, for sure, no problem. Checked usages and seems like this method used nowhere. Maybe delete it for now?